front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -148,6 +153,23 @@ const ContentBuilderTopBar = ({
             </>
           )}
         </Box>
+        {!isNilOrError(locales) && selectedLocale && locales.length > 1 && (
+          <Box
+            borderLeft={`1px solid ${colors.separation}`}
+            borderRight={`1px solid ${colors.separation}`}
```

Comment:
Not applicable for this PR. I thought I'd mention that here as I might forget. It might be a good candidate to add `borderX` and `borderY` short hand in Box component in future in another PR. Something similar to https://styled-system.com/api/

Reply:
Yes, good idea! We do something similar with the margin and padding props but not for borders. Perhaps you can add it, when you get a chance?